cachito/web/migrations/versions/c6ac095d8e9f_add_gomod_vendor_check_flag.py

```diff
@@ -0,0 +1,43 @@
+"""Add the gomod-vendor-check flag
```

Comment:
It was autogenerated with `cachito db migrate`, right?

Reply:
Autogenerated, renamed and edited, yeah. The title, `flag_table`, `upgrade()` and `downgrade()` were edited manually.